src/components/SearchForm/_search-form.scss

```diff
@@ -31,13 +31,20 @@
 }
 
 .search-form__btn-submit {
-  background-color: var(--colour-blue-50);
+  background-color: var(--colour-blue-60);
   border-bottom-left-radius: 0;
   border-top-left-radius: 0;
-  flex-basis: calc(4.375 * var(--space-unit));
+  flex-shrink: 0;
   height: auto;
   justify-content: center;
-  transition: opacity ease calc(0.5 * var(--transition-duration));
+
+  &:hover {
```

Comment:
Just a question: do we have a global `focus` state and style for it or we are using the browser default?

Reply:
good point - the :focus needs adding!